ftw/simplelayout/browser/resources/ftw.simplelayout.js

```diff
@@ -9319,7 +9323,7 @@ define(function (require, exports, module) {
 
 },{"amdefine":13}],58:[function(require,module,exports){
 /*!
- * EventEmitter v5.1.0 - git.io/ee
+ * EventEmitter v5.2.2 - git.io/ee
```

Comment:
How is the version number related to this pull request? I don't see any changes to the code of `EventEmitter` below...